Gemfile

```diff
@@ -2,6 +2,12 @@
 
 source 'https://rubygems.org'
 
+gem 'inspec', '~> 4.17.17'
```

Comment:
An unimportant suggestion (aesthetic):

```suggestion
gem 'inspec',     '~> 4.17.17'
```

Reply:
`rubocop` doesn't like this.

Reply:
@dafyddj I surrender!

Reply:
@dafyddj Separate question: what fails with `4.18.0`?

Reply:
`rspec-retry` in a similar way as happened with `describe.one` - they are clearly making some changes upstream but nor sure if they're catching all the use cases.

Reply:
I have a feeling that we should comment these issues in the `Gemfile` (likewise the one in `vault-formula`).  It will help us (or the next person) to figure out what needs to be checked when addressing this in the future.

Reply:
Good idea

Reply:
As informed in Slack, `inspec` is working now with the latest version, so this restriction is no longer in place.